public/app/plugins/datasource/loki/querybuilder/parsing.ts

```diff
@@ -267,7 +277,7 @@ function getJsonExpressionParser(expr: string, node: SyntaxNode): QueryBuilderOp
   const parserNode = node.getChild(Json);
   const parser = getString(expr, parserNode);
 
-  const params = [...getAllByType(expr, node, JsonExpression)];
+  let params = [...getAllByType(expr, node, JsonExpression)];
```

Comment:
We are not changing params here, so should it be `const` as previously?

Reply:
Oh, thanks - good catch!